reviews/models/mixins.py

```diff
@@ -177,6 +177,7 @@ def update_last_transitioned(self, ev):
     def save_changes(self, ev):
         now = self.action.date_created if self.action is not None else timezone.now()
         should_publish = self.reviewable.in_public_reviews_state
+        self.reviewable.node._has_abandoned_preprint = False
```

Comment:
Minor: to avoid potentially repeating queries, save `self.reviewable.node` to a variable.

```python
node = self.reviewable.node
node._has_abandones_preprint = False

....
node.save()
```